build.sbt

```diff
@@ -827,11 +827,46 @@ lazy val baseEngineRuntime = (project in engine("base/runtime")).
     },
   ).dependsOn(baseEngineApi, interpreter, testUtil % "test")
 
+lazy val nuKafkaEngineBinTest: Project = (project in engine("base/nu-kafka-engine-bin-test")).
+  configs(IntegrationTest).
+  settings(itSettings()).
+  enablePlugins().
+  settings(commonSettings).
+  settings(
+    name := "nussknacker-baseengine-runtime-kafka",
+    IntegrationTest / Keys.test := (IntegrationTest / Keys.test).dependsOn(
+      kafkaBaseEngineRuntime / Universal / stage
+    ).value,
+    libraryDependencies ++= Seq(
+      "commons-io" % "commons-io" % commonsIOV
+    )
+  ).dependsOn(interpreter % "it", kafkaUtil % "it", testUtil % "it", kafkaTestUtil % "it")
+
 
-lazy val kafkaBaseEngineRuntime = (project in engine("base/kafka")).
+lazy val kafkaBaseEngineRuntime: Project = (project in engine("base/kafka")).
   settings(commonSettings).
+  settings(publishAssemblySettings: _*).
+  enablePlugins(SbtNativePackager, JavaServerAppPackaging).
   settings(
-    name := "nussknacker-baseengine-runtime-kafka"
+    name := "nussknacker-baseengine-runtime-kafka",
+    assembly / assemblyOption := (assembly / assemblyOption).value.copy(includeScala = true, level = Level.Info),
```

Comment:
do we want assembly if we'll have .tgz and docker distributions?

Reply:
cleared